docs/docs/README.md

```diff
@@ -28,7 +28,7 @@ Get familiar with the SDK and explore its main concepts.
 Check out the docs for the various parts of the Cosmos stack.
 
 * [**Cosmos Hub**](https://hub.cosmos.network) - The first of thousands of interconnected blockchains on the Cosmos Network.
-* [**Tendermint Core**](https://docs.tendermint.com) - The leading BFT engine for building blockchains, powering Cosmos SDK.
+* [** CometBFT  Core**](https://docs.tendermint.com) - The leading BFT engine for building blockchains, powering Cosmos SDK.
```

Comment:
@alexanderbez - do we have 1:1 doc urls for the swap in general?

Reply:
That should be the case I believe.

Reply:
It is actually not the case, I have not verified every linked, but they are different on https://docs.cometbft.com